comtypes/server/inprocserver.py

```diff
@@ -119,7 +119,7 @@ def DllGetClassObject(rclsid, riid, ppv):
         if not cls:
             return CLASS_E_CLASSNOTAVAILABLE
 
-        result = ClassFactory(cls).IUnknown_QueryInterface(None, ctypes.pointer(iid), ctypes.c_void_p(ppv))
```

Comment:
This change is not clear to me in context of Py3 support.

Reply:
That change is from #234 that was merged.